src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
+        OrderResponseDto dto = new OrderResponseDto();
```

Comment:
Order object ) u lazy like me

Reply:
Please, don't be lazy. `object` is not a meaningful variable name